protostar/cairo/cairo_bindings.py

```diff
@@ -35,6 +35,20 @@ def compile_starknet_contract_from_path(
         )
 
 
+def compile_starknet_contract_to_sierra_from_path(
```

Comment:
Can you rename `compile_starknet_contract_from_path` => `compile_starknet_contract_to_casm_from_path`?

Since the new method indicates the output format, we should do that to the already-existing one too.